docs/ipyleaflet_example.rst

```diff
@@ -0,0 +1,127 @@
+=================
+Ipyleaflet Example
+=================
+
+Thebe can display output from ipyleaflet_ Jupyter widgets. This example_ is repurposed from the
+ipyleaflet documentation and is licensed under the BSD 3 Clause.
```

Comment:
ipyleaflet is MIT licensed

Reply:
Okay, the README and license file in their repo claim different things, but we'll go with MIT as you say and that's what the license file says.

Reply:
Oh, I might be mistaken. I just saw MIT listed from the auto detection of github. We could put whatever is in their actual license file.

Reply:
Yes we will update the README, but the license is MIT.